src/main/kotlin/io/bazel/kotlin/builder/tasks/knative/KotlinNativeTaskExecutor.kt

```diff
@@ -0,0 +1,98 @@
+package io.bazel.kotlin.builder.tasks.knative
```

Comment:
I had use to `knative` instead of `native` in the package names because `native` is a java keyword and the Dagger annotation processing on KotlinBuilder was failing due to that